src/Compilers/CSharp/Test/CommandLine/ErrorLoggerTests.cs

```diff
@@ -104,29 +105,37 @@ public class C
             }}
           ],
           ""fullMessage"": ""The field 'C.x' is never used"",
+          ""isSuppressedInSource"": false,
+          ""tags"": [
+            ""Compiler"",
+            ""Telemetry""
+          ],
           ""properties"": {{
             ""severity"": ""Warning"",
             ""warningLevel"": ""3"",
             ""defaultSeverity"": ""Warning"",
             ""title"": ""Field is never used"",
             ""category"": ""Compiler"",
-            ""isEnabledByDefault"": ""True"",
-            ""isSuppressedInSource"": ""False"",
-            ""customTags"": ""Compiler;Telemetry""
+            ""isEnabledByDefault"": ""True""
           }}
         }},
         {{
           ""ruleId"": ""CS5001"",
+          ""kind"": ""error"",
```

Comment:
"kind" is defined by SARIF. There is some possible loss of information mapping severity to "kind" so I opted to keep both.There's a comment in the code about it.